Examples/S3_Soto/Package.swift

```diff
@@ -0,0 +1,56 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+// needed for CI to test the local version of the library
+import class Foundation.ProcessInfo
+import struct Foundation.URL
+
+#if os(macOS)
+let platforms: [PackageDescription.SupportedPlatform]? = [.macOS(.v15)]
+#else
+let platforms: [PackageDescription.SupportedPlatform]? = nil
+#endif
+
+let package = Package(
+    name: "SotoLambdaExample",
+    platforms: platforms,
+    products: [
+        .executable(name: "SotoLambdaExample", targets: ["SotoExample"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/soto-project/soto.git", from: "7.0.0"),
+
+        // dependency on swift-aws-lambda-runtime is added dynamically below
+        // .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", branch: "main")
+        .package(url: "https://github.com/swift-server/swift-aws-lambda-events", branch: "main"),
+    ],
+    targets: [
+        .executableTarget(
+            name: "SotoExample",
+            dependencies: [
+                .product(name: "SotoS3", package: "soto"),
+                .product(name: "AWSLambdaRuntime", package: "swift-aws-lambda-runtime"),
+                .product(name: "AWSLambdaEvents", package: "swift-aws-lambda-events"),
+            ]
+        )
+    ]
+)
+
+if let localDepsPath = ProcessInfo.processInfo.environment["LAMBDA_USE_LOCAL_DEPS"],
```

Comment:
We could use `Context.environment` instead of `ProcessInfo.processInfo.environment` here.